opae.spec.in

```diff
@@ -16,7 +16,24 @@ Requires:       opae-libs , opae-devel , opae-tools, opae-tools-extra
 Requires:       opae-ase
 %endif
 Source0:        @CPACK_PACKAGE_NAME@.tar.gz
+URL:            https://github.com/OPAE/opae-sdk
 
+ExclusiveArch: %{ix86} x86_64
+
+
+
+BuildRequires:     gcc
+BuildRequires:     gcc-c++
+BuildRequires:     gtest-devel
+BuildRequires:     cmake
+BuildRequires:     git
+BuildRequires:     rpm-build
+BuildRequires:     python-sphinx
+BuildRequires:     python-jsonschema
+BuildRequires:     json-c-devel
+BuildRequires:     libuuid-devel 
+BuildRequires:     hwloc-libs
```

Comment:
I think this is only needed for ASE, so it should be surrounded by the %if %with_ase check.